contracts/payloads/IGP113/PayloadIGP113.sol

```diff
@@ -0,0 +1,484 @@
+                    supplyToken: OSETH_ADDRESS,
+                    borrowToken: address(0), // Set at DEX level
+                    baseWithdrawalLimitInUSD: 7_000, // $7k
+                    baseBorrowLimitInUSD: 0,
+                    maxBorrowLimitInUSD: 0
+                });
+                setVaultLimits(VAULT_OSETH_USDC_USDT);
+                VAULT_FACTORY.setVaultAuth(
+                    OSETH_USDC_USDT_VAULT,
+                    TEAM_MULTISIG,
+                    true
+                );
+            }
+
+            {
+                // Set borrow limits at DEX level for TYPE_3 vault
+                DexBorrowProtocolConfigInShares
+                    memory config_ = DexBorrowProtocolConfigInShares({
+                        dex: USDC_USDT_DEX,
+                        protocol: OSETH_USDC_USDT_VAULT,
+                        expandPercent: 30 * 1e2, // 30%
+                        expandDuration: 6 hours, // 6 hours
+                        baseBorrowLimit: 3500 * 1e18, // 3500 shares or $7k
+                        maxBorrowLimit: 4500 * 1e18 // 4500 shares or $9k
+                    });
+                setDexBorrowProtocolLimitsInShares(config_);
+            }
+        }
+
+        // Vault ID 157: OSETH / USDC-USDT concentrated (TYPE_3) - Dust limits
+        {
+            address OSETH_USDC_USDT_CONC_VAULT = getVaultAddress(157);
+            address USDC_USDT_DEX = getDexAddress(2);
+
+            {
+                VaultConfig memory VAULT_OSETH_USDC_USDT_CONC = VaultConfig({
+                    vault: OSETH_USDC_USDT_CONC_VAULT,
+                    vaultType: VAULT_TYPE.TYPE_3,
+                    supplyToken: OSETH_ADDRESS,
+                    borrowToken: address(0), // Set at DEX level
+                    baseWithdrawalLimitInUSD: 7_000, // $7k
+                    baseBorrowLimitInUSD: 0,
+                    maxBorrowLimitInUSD: 0
+                });
+                setVaultLimits(VAULT_OSETH_USDC_USDT_CONC);
+                VAULT_FACTORY.setVaultAuth(
+                    OSETH_USDC_USDT_CONC_VAULT,
+                    TEAM_MULTISIG,
+                    true
+                );
+            }
+
+            {
+                // Set borrow limits at DEX level for TYPE_3 vault
+                DexBorrowProtocolConfigInShares
+                    memory config_ = DexBorrowProtocolConfigInShares({
+                        dex: USDC_USDT_DEX,
+                        protocol: OSETH_USDC_USDT_CONC_VAULT,
+                        expandPercent: 30 * 1e2, // 30%
+                        expandDuration: 6 hours, // 6 hours
+                        baseBorrowLimit: 3500 * 1e18, // 3500 shares or $7k
+                        maxBorrowLimit: 4500 * 1e18 // 4500 shares or $9k
+                    });
+                setDexBorrowProtocolLimitsInShares(config_);
+            }
+        }
+
+        // Vault ID 158: oseth-eth <> wsteth-eth (TYPE_2) - Dust limits
+        {
```

Comment:
this is a type 4 on WSTETH-ETH dex id 1borrow side. supply side we can set on dex ourselves via team MS